src/components/common/SceneWrapper.tsx

```diff
@@ -38,6 +44,14 @@ interface Props {
   // True if this scene has a bottom tab bar:
   hasTabs?: boolean
 
+  // True if we want to show the notification view:
+  hasNotifications?: boolean
+
+  // If we want the SceneWrapper's ScrollView to overscroll if it
+  // hasNotifications. Defaults to true. If false, means the scene MUST manage
+  // overscrolls using the passed 'gap':
+  hasOverscroll?: boolean
+
```

Comment:
No we still need this to allow fine tuning where the caller wants to have full
control over overscroll.

We want a specific set of scenes with hasNotifications, but for some of those
scenes, we will disable hasOverscroll so that the overscroll is managed by the
calling scene. For all other cases (hasOverscroll), the overscroll is managed
from SceneWrapper.